website/docs/docs/supported-data-platforms.md

```diff
@@ -60,6 +38,29 @@ Who made and maintains and adapter is certainly relevant, but we recommend using
 | [AWS Glue](glue-setup)                     | [Oracle](oracle-setup)           |                                      |
 | [Greenplum](greenplum-setup)               | [Rockset](rockset-setup)         |                                      |
 
+## Adapter Installation
+
+With a few exceptions [^1], all adapters listed below can be installed from PyPI using `pip install <ADAPTER-NAME>`. The installation will include `dbt-core` and any other required dependencies, which may include both other dependencies and even other adapter plugins. Read more about [installing dbt](/docs/get-started/installation).
+
+## Adapter Taxonomy
+
+### Verified by dbt Labs
+
+In order to provide a more consistent and reliable experience, dbt Labs now has a rigorous process by which we verify adapter plugins. The process covers aspects of development, documentation, user experience and maintenance. These adapters earn a "Verified" designation so that users can have a certain level of trust and expectation when they use them. To learn more see [Verifying a new adapter](/guides/advanced/adapter-development/7-verifying-a-new-adapter).
```

Comment:
```suggestion
In order to provide a more consistent and reliable experience, dbt Labs has a rigorous process by which we verify adapter plugins. The process covers aspects of development, documentation, user experience, and maintenance. These adapters earn a **Verified** designation so that users can have a certain level of trust and expectation when they use them. To learn more, see [Verifying a new adapter](/guides/advanced/adapter-development/7-verifying-a-new-adapter).
```